nick-fury/scrape.py

```diff
@@ -0,0 +1,81 @@
+import json
+import requests
+import datetime
+import MySQLdb
+import time
+
+def strip(s):
+	return s.strip().lower()
+
+def task(con1,cur1,con2,cur2):
+	print("Task started")
+
+	start =  time.time()
+	url = "https://api.tfl.gov.uk/bikepoint"
+	response = requests.get(url)
+	data = response.text
+	data = json.loads(data)
+
+	for i in range(len(data)):
+		try:
+			terminal = {}
+			terminal['id']=data[i]['id']
+			terminal['commonName']=" ".join(list(map(strip,data[i]['commonName'].split(','))))
+			
+			info = {}
+			for j in data[i]['additionalProperties']:
+				if(j['key']=='TerminalName'):
+					terminal['terminalName']=j['value']
+					info['terminalName']=j['value']
+				elif(j['key']=='NbBikes'):
+					info['nbBike']=j['value']
+				elif(j['key']=='NbEmptyDocks'):
+					info['nbEmptyDock']=j['value']
+				elif(j['key']=='NbDocks'):
+					info['nbDock']=j['value']
+
+			d = data[i]['additionalProperties'][6]['modified'].split('T')

```

Comment:
You're assuming the date will always come at the 6th element, what if that changes?

Reply:
I have resolved this issue.